.github/workflows/build_wheels.yml

```diff
@@ -3,6 +3,23 @@ name: Build
 on: [pull_request]
 
 jobs:
+  make_sdist:
+    name: Make SDist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Build SDist
+        run: pipx run build --sdist
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: cibw-sdist
+          path: dist/*.tar.gz
+
+      - name: Test if SDist build has all necessary pieces to be installed
```

Comment:
It would be good to add some actual tests in the future. Maybe just some basic importing modules and other processes that rely on non-python files (SQL, sqlite, zip, etc)